documentation/release_notes.rst

```diff
@@ -8,6 +8,51 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.9.0
+===============
+
+New Features
+------------
+- Added parallel range algorithms in ``namespace oneapi::dpl::ranges``: ``fill``, ``move``, ``replace``, ``replace_if``,
+``remove``, ``remove_if``, ``mismatch``, ``minmax_element``, ``min``, ``max``, ``find_first_of``, ``find_end``, ``is_sorted_until``
+- Improved performance of set operation algorithms (e.g., ``set_union``, ``set_difference``, etc.)
+
+Fixed Issues
+------------
+- Eliminated runtime exceptions that were encountered when compiling code that uses device policies with the open source
+oneAPI DPC++ Compiler (clang++ driver) without specifying an optimization flag.
```

Comment:
One may think that the whole device backend was broken with that compiler with device policies. Let's make the note more specific. @mmichel11, could you make a suggestion?

Reply:
This was affecting `scan` based algorithms.

I would phrase it, "when using device policies with ``inclusive_scan``, ``copy_if``, ``partition``, ``unique``, ``reduce_by_segment``, and related algorithms with the open source..."

Reply:
@mmichel11 what are the "and related algorithms"?

Reply:
By related algorithms, I mean the other scan-based algorithms in the same family: things like `exclusive_scan`, `remove_if`, `partition_copy`, `unique_copy`, etc.

Reply:
I've made the suggested change.